django/forms/forms.py

```diff
@@ -266,15 +267,20 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
                 output.append(str_hidden)
         return mark_safe('\n'.join(output))
 
+    def bound_fields(self):
+        for name, field in self.fields.items():
+            yield self[name]
+
+    def get_context(self):
+        return {'form': self}
+
     def as_table(self):
```

Comment:
We'd likely want to accept `renderer=None` here and default to `get_default_renderer()`.